frappe/desk/page/setup_wizard/setup_wizard.js

```diff
@@ -25,21 +28,24 @@ frappe.wiz = {
 frappe.pages['setup-wizard'].on_page_load = function(wrapper) {
 	// setup page ui
 	$(".navbar:first").toggle(false);
-	$("body").css({"padding-top":"30px"});
+
+	$('header').append(`<div class="setup-wizard-brand"">
+		<img src="/assets/frappe_theme/img/erp-icon.svg" class="brand-icon"
```

Comment:
@pratu16x7,

dependency on frappe_theme app !!

Also, ERPNext logo should not be included in frappe app, may be we can extend the icon if erpnext app is not installed then show only frappe icon